src/modules/fetcher/__tests__/FetcherModule.spec.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import FetcherModule from '../FetcherModule'
+import NativeFetcherModule from '../NativeFetcherModule'
+import type { TargetFilePathsType } from '../FetcherModule'
+
+jest.mock('../NativeFetcherModule')
+
+beforeEach(() => {
```

Comment:
I guess it's better to expect a clean environment and clean it afterwards.
```suggestion
afterEach(() => {
```

Reply:
In case of resetAllMocks() I'd actually prefere to run it before each test. That way you can be sure that the function mocks are all reset for the very first test.

Note that resetAllMocks only affects `jest.fn` mocks